Source/WebKit/UIProcess/Automation/BidiBrowserAgent.cpp

```diff
@@ -80,6 +81,10 @@ Inspector::CommandResult<void> BidiBrowserAgent::close()
 
     session->terminate();
 
+    // FIXME: this should not depend on a particular process pool.
+    if (auto* client = session->protectedProcessPool()->automationClient())
+        client->closeBrowser();
```

Comment:
We could return the `"unable to close browser"` error if client is not set, with an error message like "Could not find automation client" or similar. From the spec:

> "unable to close browser": Tried to close the browser, but failed to do so.

Reply:
Done.